package.json

```diff
@@ -25,20 +25,20 @@
   "dependencies": {},
   "devDependencies": {
     "doctoc": "^2.0.0",
-    "eslint": "^7.23.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-eslint-plugin": "^3.0.0",
-    "eslint-plugin-jest": "^24.3.4",
+    "eslint": "^8.1.0",
```

Comment:
Sticking to ESLint v7 in the dev dependencies would unblock this PR could since the main changes for adding ESLint v8 compatibility to the package are already addressed in this very PR. Upgrading ESLint here is just for the local environment used to develop the project, that must not block this PR to add ESLint v8 compatibility to the package.